### PR TITLE
🎨 improve tip e2e test

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- 🎨 improve tip e2e test
  - Our end-to-end tests are waiting to verify whether all progress bars report 100%.
  - This is not always the case (I’ve spent some time investigating, but I don’t know the reason yet).
  - Nevertheless, the frontend doesn't care. Once everything is ready and downloaded, the service starts. Therefore, I modified the test so that, in cases where 100% is not achieved, it will still proceed to the next step, just as the frontend would.

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py [hunk 1]
```diff
@@ -327,10 +327,18 @@ def expected_service_running(
...
-        with websocket.expect_event("framereceived", waiter, timeout=timeout):
-            if press_start_button:
-                _trigger_service_start(page, node_id)
-            yield service_running
+
+        try:
+            with websocket.expect_event("framereceived", waiter, timeout=timeout):
+                if press_start_button:
+                    _trigger_service_start(page, node_id)
+        except Exception:
```
Comment (pcrespov) [inline packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py]: how do you know this timeout comes from the `with websocket.expect_event`? Note that `_trigger_service_start` also incorporates a timeout in the implementation!

File: packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py [hunk 2]
```diff
@@ -327,10 +327,18 @@ def expected_service_running(
...
+            ctx.logger.warning(
+                "⚠️ Progress bar didn't recieve 100 percent: %s ⚠️",
```
Comment (pcrespov) [inline packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py:337]: "receive"

File: packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py [hunk 3]
```diff
@@ -327,10 +327,18 @@ def expected_service_running(
...
-        with websocket.expect_event("framereceived", waiter, timeout=timeout):
-            if press_start_button:
-                _trigger_service_start(page, node_id)
-            yield service_running
+
+        try:
+            with websocket.expect_event("framereceived", waiter, timeout=timeout):
+                if press_start_button:
+                    _trigger_service_start(page, node_id)
+        except TimeoutError:
```
Comment (pcrespov) [inline packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py:335]: How do you guarantee that it was raised by  `websocket.expect_event` and not raised by `_trigger_service_start`
Moreover, that the cause of the timeout is the progress and not something else?

TIP:
- create `SocketIONodeProgressCompleteWaiter.last_progress` 
- check after the exception `if waiter.last_progress`
- otherwise let the exception raise

File: packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py (hunk 2 above)
Comment (pcrespov) [reply packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py:337]: Alsp please add a reference to this PR (e.g. url) to understand in the future what lead us to ignore this check

**Review by pcrespov — APPROVED** [review]
👍

**Review by mrnicegyu11 — APPROVED** [review]
tldr lgtm as discussed :1st_place_medal:
